scripts/test

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+command="$*"
+project="ffc-template-node"
+service="ffc-template-node"
+
+compose() {
+  docker-compose \
+    -f docker-compose.yaml \
+    -f docker-compose.test.yaml \
+    -p "$project-test" \
+    "$@"
+}
+
+(
+  cd "$projectRoot"
+
+  # Guarantee clean environment
+  compose down --volumes
+
+  # Ensure container images are up to date
+  compose build
+
+  # Run tests
+  compose run "$service" "$command"
+
+  # Clean up
+  compose down --volumes
```

Comment:
This is something we have in our current demo repos and it's something about our setup I don't like.  It's not a problem with the script itself, but more how we use compose files and what will happen as a result of the script.

Using the [claim service compose file](https://github.com/DEFRA/ffc-demo-claim-service/blob/master/docker-compose.yaml), you'll see the postgres service is defined here so it is used by both local development and the tests.

That means they are sharing a volume, so if during my local development I add a load of test data to this service and then want to run the tests through this script, I will lose all of that data and have to keep adding it again between test runs which is going to frustrate developers.

A better way in my opinion is to have separate Postgres services defined in the `override` and `test` compose files, then this script won't affect my local development database as they have independent data volumes.  This is the approach I've followed elsewhere, example in this [override file](https://github.com/johnwatson484/dream-league-api/blob/master/docker-compose.override.yaml) and related [test file](https://github.com/johnwatson484/dream-league-api/blob/master/docker-compose.test.yaml)

If teams don't want to do that, then fine it's their call but this script is quite destructive if not.